### PR TITLE
HTM School content update

### PR DESCRIPTION
**Pull request by rhyolight**

### CODE REVIEW

File: packages/numenta.com/pages/htm-school/index.jsx [hunk 1]
```diff
@@ -162,7 +162,7 @@ const SchoolPage = (props, {config}) => {
           <div className={styles.content}>
             <Paragraph>
               An SDR is simply a list of bits, each bit being
-              <code>0</code>
+              {' '} <code>0</code> {' '}
               or
               <code>1</code>.
```
Comment (lscheinkman) [inline packages/numenta.com/pages/htm-school/index.jsx]: I am confuse about all those `{' '}`. It seems arbitrary. Why line 165 and not 167?

File: packages/numenta.org/pages/_Section.jsx [hunk 2]
```diff
@@ -157,7 +157,7 @@ const SectionHome = (props, {config}) => {
             Github organization
           </TextLink> {' '}
           . If you're new to HTM, you'll learn a lot in {' '}
-          <TextLink to={links.out.school}>
+          <TextLink to={links.in.school}>
```
Comment (rhyolight) [inline packages/numenta.org/pages/_Section.jsx]: Why do I keep changing this?

**Comment by rhyolight** [comment]
@lscheinkman Sorry about the mess, I'm still learning the best way.

File: packages/numenta.com/pages/htm-school/index.jsx (hunk 1 above)
Comment (rhyolight) [inline packages/numenta.com/pages/htm-school/index.jsx]: I'm confused about this, too. There are places in the output where some react components (words) render without a space next to other words. I've been putting `{' '}` in those places there should be spaces as-needed.

Comment (rhyolight) [inline packages/numenta.com/pages/htm-school/index.jsx]: In this case, I did not want a space before the `.`. I am not sure why I needed to inject a space before the first `<code>` block, but not the 2nd.

**Pull request closed without merging**